extensions/wikia/AppPromoLanding/AppPromoLandingController.class.php

```diff
@@ -139,16 +148,16 @@ public function content( $params ){
 						//'url' => $item['url'],
 						'title' => $item['title'],
 						'imgUrl' => $thumbnail,
-						'width' => self::IMG_WIDTH,
-						'height' => self::IMG_HEIGHT
+						'width' => static::IMG_WIDTH,
+						'height' => static::IMG_HEIGHT
 					];
 				}
 			}
 		}
 
 		// Not all articles will have images, so we may have more or less than we need. Here,
 		// we will right-size the array.
-		$numThumbsNeeded = ( self::THUMBS_NUM_ROWS * self::THUMBS_PER_ROW );
+		$numThumbsNeeded = ( static::THUMBS_NUM_ROWS * static::THUMBS_PER_ROW );
 		if(count( $trendingArticles ) > $numThumbsNeeded){
 			$trendingArticles = array_slice($trendingArticles, 0, $numThumbsNeeded);
 		} else if(count( $trendingArticles ) < $numThumbsNeeded ){
```

Comment:
What do you think about using `array_rand` in this condition: http://php.net/manual/en/function.array-rand.php ?